tools/pnnx/src/CMakeLists.txt

```diff
@@ -436,7 +437,6 @@ set(pnnx_pass_ncnn_SRCS
     pass_ncnn/F_max_pool1d.cpp
     pass_ncnn/F_max_pool2d.cpp
     pass_ncnn/F_max_pool3d.cpp
-    pass_ncnn/F_mish.cpp
```

Comment:
fix this oops

Reply:
I move to test to conditional section like F.mish and fix this oops, but it still have checks were not successful